core/src/test/java/org/apache/gravitino/storage/TestEntityStorage.java

```diff
@@ -151,14 +183,16 @@ private void destroy(String type) {
       } catch (Exception e) {
         // Ignore
       }
-    } else if (type.equals(Configs.RELATIONAL_ENTITY_STORE)) {
+    } else if (type.equalsIgnoreCase("h2") || type.equalsIgnoreCase("mysql")) {
       dropAllTables();
       File dir = new File(DB_DIR);
       if (dir.exists()) {
         dir.delete();
       }
 
       FileUtils.deleteQuietly(new File(H2_FILE));
+    } else if (type.equalsIgnoreCase("postgresql")) {
+      // Do nothing
```

Comment:
Why doesn't the PG backend need to clear existing resources like tables and files?

Reply:
PG has the concept database and schema, and each catalog uses a different database name. For PostgreSQL catalog:

The database in PosgreSQL is equivalent to the Gravitino catalog, and the schema is equivalent to the Gravitino  database.